changelog.d/gh-8365.added

```diff
@@ -0,0 +1,9 @@
+feat(naming): Adding support for naming propagation when the left-hand side (lhs) of a variable definition is an identifier pattern
```

Comment:
```suggestion
Added support for naming propagation when the left-hand side (lhs) of a variable definition is an identifier pattern
```

We settled on not using the conventional commit style for changelog entries, I believe. We had a recent slack discussion on this. We also typically use the past tense in changelog entries.